src/components/PaginatedTable/PaginatedTable.stories.mdx

```diff
@@ -0,0 +1,72 @@
+# PaginatedTable Component
+
+## Type: Composite
+
+The PaginatedTable contains the following components: [Table](https://design-system.codelitt.dev/?path=/docs/components-table--default), [Pagination](https://design-system.codelitt.dev/?path=/docs/components-pagination--default) and [Input](https://design-system.codelitt.dev/?path=/docs/components-numeric-input--default).
```

Comment:
@branquinhoaa please update the documentation. We're using the NumericInput component, not the default Input. Let's be clear and concise on which components (even if they're decorators) to track potential issues.